coretex/entities/task_run/parameter/utils.py

```diff
@@ -30,3 +30,18 @@ def validateEnumStructure(name: str, value: Optional[Any], required: bool) -> Tu
         return False, f"Enum parameter \"{name}.selected\" has invalid type. Expected \"int\", got \"{type(selected).__name__}\""
 
     return True, None
+
+
+def validateRangeStructure(name: str, value: Dict[str, Any], required: bool) -> Tuple[bool, Optional[str]]:
+    if len(value) != 3 or "from" not in value or "to" not in value or "step" not in value:
+        keys = ", ".join(value.keys())
+        return False, f"Range parameter \"{name}\" must contain only \"from\",  \"to\" and \"step\" properties, but it contains \"{keys}\""
+
+    if any(element is None for element in value.values()) and required:
+        return False, f"Elements of range parameter \"{name}\" must not be null"
+
+    if not all(isinstance(element, int) for element in value.values()) and required:
```

Comment:
`float` and `bool` are a subclasses of `int` in Python so `isintance(element, int)` will return `True` for cases where `element` is either a `float` or a `bool`. Can this be changed to check if `element` is `int` but not `int` subclasses?